### ISSUE TITLE
issue with affine transform

### ISSUE THREAD

**Issue by sibajigaj** [fepegar/torchio#141]
Hi, 
    Thanks for the nice augmentation tool. I am trying to apply on our dataset where the image size is 160 X 384 X 384 and voxel spacing is .7 X .36 X.36 mm. The error is coming for affine transform. The error is as follows:

```python-traceback
Traceback (most recent call last):
  File "test3.py", line 50, in <module>
    transformed=transform(sample)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/transform.py", line 34, in __call__
    sample = self.apply_transform(sample)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/augmentation/composition.py", line 28, in apply_transform
    return self.transform(sample)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchvision/transforms/transforms.py", line 61, in __call__
    img = t(img)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/augmentation/random_transform.py", line 33, in __call__
    return super().__call__(sample)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/transform.py", line 34, in __call__
    sample = self.apply_transform(sample)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/augmentation/spatial/random_affine.py", line 105, in apply_transform
    interpolation,
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/augmentation/spatial/random_affine.py", line 167, in apply_affine_transform
    default_value = get_borders_mean(image, filter_otsu=True)
  File "~/software/virtual/pytorch/lib64/python3.6/site-packages/torchio/transforms/augmentation/spatial/random_affine.py", line 195, in get_borders_mean
    array[:, :, -1],
ValueError: could not broadcast input array from shape (384,160) into shape (384). 
```

The affine transform is working for the given MRI image dataset  in TorchIO tutorial.ipynb tutorial. Also it is working if I pass default_pad_value='minimum' in the RandomAffine function. What could be possible processing, I need to do before doing affine transform ?

**Comment by fepegar** [comment]
Hi @sibajigaj,

What version of TorchIO are you using?

**Comment by sibajigaj** [comment]
Version: 0.16.4

**Comment by fepegar** [comment]
Could you please share the code to reproduce this and, if possible, an image? You can send it to fernando.perezgarcia.17@ucl.ac.uk, if you like.

**Comment by fepegar** [comment]
Ok I just reproduced it. I'll fix that in a moment.

**Comment by fepegar** [comment]
Fixed in `0.16.6`. Thanks for reporting, @sibajigaj.
